cluster/examples/vm-windows2012r2.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# A windows VM leveraging the Windows 2k12r2 preset
+#
+apiVersion: kubevirt.io/v1alpha2
+kind: VirtualMachine
+metadata:
+  name: windows
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/os: win2k12r2
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: sata
+            name: pvcdisk
+            volumeName: pvcvolume
+          interfaces:
+          - bridge: {}
+            model: e1000
```

Comment:
@booxter so, in testing we see that the model will not appear in the vmi when we start the vm.

Do you know why?

Reply:
No I don't. Perhaps a bug or an old release you are using?

Reply:
Thanks. Was my env. I setup 0.7.0 cleanly and everything works as expected.

See https://github.com/kubevirt/kubevirt/issues/1337